### PR TITLE
[ACS} Send customer displayname - TI api

### PR DESCRIPTION
**Pull request by aidhage**

### CODE REVIEW

File: src/Common/Constants.ts [hunk 1]
```diff
@@ -12,4 +12,5 @@ export default class Constants {
   public static readonly defaultLocale = "en-us";
   public static readonly noContentStatusCode = 204;
   public static readonly transactionid = "transaction-id";
+  public static readonly customerDisplayName = "customerDisplayName"
```
Comment (krishna0209) [inline src/Common/Constants.ts:15]: Semicolon?

File: src/SDK.ts [hunk 2]
```diff
@@ -972,15 +973,19 @@ export default class SDK implements ISDK {
...
     // avoiding logging Info for typingindicator to reduce log traffic
     const timer = Timer.TIMER();
+    const { customerDisplayName } = sendTypingIndicatorOptionalParams;
     if (!currentLiveChatVersion || currentLiveChatVersion !== LiveChatVersion.V2) { throw new Error('Typing indicator is only supported on v2') }
     const endpoint = `${this.omnichannelConfiguration.orgUrl}/${OmnichannelEndpoints.SendTypingIndicatorPath}/${requestId}`;
     const axiosInstance = axios.create();
 
     const headers: StringMap = Constants.defaultHeaders;
     headers[OmnichannelHTTPHeaders.organizationId] = this.omnichannelConfiguration.orgId;
+    if(customerDisplayName){
```
Comment (krishna0209) [inline src/SDK.ts:986]: space after if (

**Review by krishna0209 — APPROVED** [review]
I assume these changes aren't backward compatible? This is ok as long as you have validated that in IC3 mode the TI API isn't called.

**Comment by aidhage** [comment]
> I assume these changes aren't backward compatible? This is ok as long as you have validated that in IC3 mode the TI API isn't called.
It should be backward compatible, I am passing customername as optional param

File: src/SDK.ts (hunk 2 above)
Comment (xTEddie) [reply src/SDK.ts:986]: after { as well

**Review by xTEddie — APPROVED** [review]

**Pull request merged**